0028-web-monetization/0028-web-monetization.md

```diff
@@ -41,33 +41,37 @@ Web Monetization makes use of [SPSP](../0009-simple-payment-setup-protocol/0009-
   
   )
-  - The browser will generate a fresh UUID (version 4) and use this as the Monetization ID from this point forward. 
-- The browser passes these payment details to the user's provider (see [Payment Handler Flow](#payment-handler-flow)).
-- The provider resolves the payment pointer and begins to pay. The provider MAY be acting from a different machine from the user. Cookies and session information MUST NOT be carried with any requests made to resolve the Payment Pointer, with the exception of the Monetization ID.
-  - On the SPSP query to resolve the Payment Pointer, a `Web-Monetization-Id` header is sent, containing the Monetization ID. The server running the web-monetized site may use this to associate future requests by the user with their payments.
-  - With the `destination_account` and `shared_secret` fetched from the SPSP query, a STREAM connection is established. A single STREAM is opened on this connection, and a positive SendMax is set.
-  - The provider SHOULD set their SendMax high enough that it is never reached, and make payment adjustments by limiting throughput.
-- Once the STREAM connection has fulfilled an ILP packet with a non-zero amount, the provider notified the browser, and the browser dispatches an event on `document.monetization`. Payment SHOULD continue.
-  - The user's agent sets `document.monetization.state` to `started`. This MUST occur before the `monetizationstart` event is fired.
-  - The event's type is `monetizationstart`. The event has a `detail` field with an object containing the Payment Pointer and the Monetization ID ([specified below](#monetizationstart)).
-  - The user's agent also emits a `monetizationprogress` ([specified below](#monetizationprogress)) event from `document.monetization`, corresponding to this first packet. If there are no listeners the event MAY NOT be emitted.
-- Payment continues for the lifetime of a given meta tag (or semantically equivalent)
-  - The provider MAY decide to stop/start payment at any time, e.g. if the user is idle, backgrounds the page, or instructs the browser to do so.
-  - If the STREAM connection is severed, the provider will redo the SPSP query to the same Payment Pointer as before with the same Monetization ID. The browser MUST NOT re-process the `<meta>` tags.
-  - Each time a packet with a nonzero amount is fulfilled, the provider notifies the browser, and the browser emits an event on `document.monetization`. The event's type is `monetizationprogress`. The event has a `detail` field containing the details of the packet ([specified below](#monetizationprogress)). If there are no listeners the event MAY NOT be emitted.
-  - When a stream is closed the `document.monetization.state` MUST be set back to 'pending'
+
+  - The user's agent generate a fresh **random** UUID (version 4) and uses this as a Monetization ID. This MUST be regenerated upon dynamic changes to the meta tags and MUST be unique per page load in order to avoid user tracking.
```

Comment:
I like this, I'm not sure if we should put in the to prevent user tracking, @sharafian curious about your thoughts on this.

Reply:
it's fine to call it out explicitly. There's a grammatical error though: `generate a fresh random UUID` should be `generates a fresh random UUID`